### PR TITLE
Fix the nit in the overview

### PR DESCRIPTION
**Pull request by timifasubaa**
First joeynmt PR :)

It could also be "And then the". Lmk which you prefer

@juliakreutzer

### CODE REVIEW

**Comment by timifasubaa** [comment]
Actually, how about `The manager receives a model and the he following steps are done:configuration as input and sets up the logger, the learning rate scheduling, the optimizer, counters for update steps, keeps track of the currently best checkpoint and decides when to stop training.` -> `The manager receives a model and then performs the following steps: 
parses the input configuration, sets up the logger, schedules the learning rate, sets up the optimizer and counters for update steps. It then keeps track of the current best checkpoint to determine when to stop training.`

**Comment by juliakreutzer** [comment]
Yes! Thank you @timifasubaa for the correction! Please go ahead and update the file as you proposed :+1:

**Comment by timifasubaa** [comment]
@juliakreutzer done :)
